masterlist.yaml

```
@@ -2799,6 +2799,23 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 7
 
+  - name: '(One of a Kind Weapons|OoaKW - Staff of Worms)?\.esp'
```

The regex should just catch one or the other, so the `?` is unnecessary.

```suggestion
  - name: '(One of a Kind Weapons|OoaKW - Staff of Worms)\.esp'
```